docs/src/kit-docs/index.ts

```diff
@@ -1 +0,0 @@
-export { Tabs, TabPanel, TabItem } from './tabs'
```

Comment:
Do we still need this file or parent folder?

Reply:
yes to the parent folder (to be able to use Tabs in the .md files that are the docs pages), no to the index.ts file.